### DISCUSSION TITLE
Add no stopping between profile changes

### DISCUSSION THREAD

**Discussion by UrmasK222** [Rem0o/FanControl.Releases#900] [Ideas]
Hi

Awesome soft! Thank you!

Currently when I change profiles, fans are stopped completely for a moment. Maybe add some kind of a "crossfade" between profiles? Then the fans would never stop. 

Br
Urmas

**Comment by Rem0o** [comment]
What happens when you turn off a control card? Does it stop? 
For the few computers I've used, the transition is smooth between profiles. It goes from whatever % it was to the % the next profile dictates.

**Reply by UrmasK222** [reply]
Thank you for the quick reply. 

Help me to understand what is a control card? Not sure I understand what you mean. This is how it looks now with my silent profile.

![image](https://user-images.githubusercontent.com/45017739/176832922-6dd74e36-e235-4c7e-972f-2f4c49850b78.png)

**Reply by Rem0o** [reply]
![image](https://user-images.githubusercontent.com/14118956/176933873-4f98af5d-c1c7-4993-a9ef-a6f5ef1ca02e.png)

**Comment by UrmasK222** [comment]
TBH that's not an distracting or disturbing issue for me at all.  

I did some tests. Cards off switches fans & pumps to bios profiles.

What I've found out. 

**First (check the image)**
1) Full speed all
2) Turn off all cards
3) Applied a different profile
4) applied "Full speed" profile"
5) Applied a different profile (now the step down works)
![image](https://user-images.githubusercontent.com/45017739/177011397-9f213e0a-fa43-48ff-9e5b-d2cd193c30c8.png)


**Secondly**
I created one profile where cards are turned off. 

1) Turn on full speed profile
2) Turn on "Cards off" profile
No stopping between 

**Third**
I switched from "Cards off" profile to "normal profile"
![image](https://user-images.githubusercontent.com/45017739/177011438-8483993c-7623-4279-8fb8-3dca08a326e5.png)

**Fourth**
I switched from normal to cards off
![image](https://user-images.githubusercontent.com/45017739/177011479-4e663914-b626-4baf-a5b6-30f7ec8627cc.png)
